src/scripts/main.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+document.addEventListener('click', el => {
```

Comment:
There's no need to add event listener to the whole document, only wall should have it.

Reply:
That's a clever solution. Thank you!)